code/controllers/subsystem/traumas.dm

```diff
@@ -158,25 +160,29 @@ SUBSYSTEM_DEF(traumas)
 						
 					"cats" = typecacheof(list(/obj/item/organ/ears/cat, /obj/item/organ/tail/cat, /obj/item/laser_pointer, /obj/item/toy/cattoy, /obj/item/clothing/head/kitty,
 						/obj/item/clothing/head/collectable/kitty, /obj/item/melee/chainofcommand/tailwhip/kitty, /obj/item/stack/sheet/animalhide/cat))
+						
+					"syndicate"   = typecacheof(list(/obj/item/stack/tile/mineral/plastitanium, /obj/item/aiModule/, /obj/machinery/recharge_station,
+						/obj/item/aicard, /obj/item/deactivated_swarmer, /obj/effect/mob_spawn/swarmer)),
```

Comment:
Not entirely sure about this one.
`/obj/item/aicard`

Reply:
place holders